java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -5769,6 +5769,140 @@ public void bzmpop_binary_timeout_check(BaseClient client) {
         }
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void xadd_duplicate_entry_keys(BaseClient client) {
+        String key = UUID.randomUUID().toString();
+        String field = UUID.randomUUID().toString();
+        String foo1 = "foo1";
+        String bar1 = "bar1";
+
+        String timestamp = "1721748920942-0";
+        String[][] entry = new String[][] {{field, foo1}, {field, bar1}};
+        assertEquals(
+                timestamp,
```

Comment:
Does it work? If yes - I'm surprised. If no, just await for `xadd` and disregard the return value.